### PR TITLE
fix(userspace/libscap): properly apply driver_event_schema target properties

### LINKED ISSUE #1201: PATCH: fix missing DSO tags for libdriver-event-schema.so
Hello. There's a bug in the build system: DSO tags for `libscap-event-schema.so` are set twice, but never for `libdriver-event-schema.so`. There should be one for each. I'm the Debian maintainer, and I just fixed this for the distro:

  https://salsa.debian.org/debian/falcosecurity-libs/-/blob/master/debian/patches/properties-for-driver-event-schema.patch

I think you should take the patch.

Thanks

### PR DESCRIPTION
**Pull request by FedeDP**
**What type of PR is this?**

/kind bug

**Any specific area of the project related to this PR?**

/area build

**Does this PR require a change in the driver versions?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #1201 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: userspace/libscap/CMakeLists.txt [hunk 1]
```diff
@@ -148,7 +148,7 @@ if(CMAKE_SYSTEM_NAME MATCHES "Linux")
 	add_library(driver_event_schema
 		../../driver/fillers_table.c)
 	target_link_libraries(scap_event_schema driver_event_schema)
-	set_scap_target_properties(scap_event_schema)
+	set_scap_target_properties(driver_event_schema)
```
Comment (FedeDP) [inline userspace/libscap/CMakeLists.txt:151]: /cc @gnosek

**Comment by gnosek** [comment]
Ah so you found it too :D I'd rather kill it with fire: https://github.com/falcosecurity/libs/pull/1217

**Pull request closed without merging**